stdpopsim/cli.py

```diff
@@ -450,6 +450,14 @@ def run_simulation(args):
                     "the developer documentation. "
                     "https://stdpopsim.readthedocs.io/en/latest/development.html"
                     "#demographic-model-review-process")
+        if args.slim_scaling_factor != 1 and not args.quiet:
```

Comment:
Shouldn't we be doing this in the ``slim_engine.py`` file? Although I see your point that it would be irritating if this raised a warning any time someone uses Q != 1. Presumably we'd want to warn when the scaling factor is extreme (but I've no idea how you'd determine this).

Reply:
I considered putting it in the slim engine simulate() method, but I'd rather not get the warning from the Python API as well as the CLI. In particular, I don't want to silence all the other potential warnings which I think are far more informative.